protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -10,6 +10,9 @@ function validateBase64(section_id, value) {
 	if (value.length != 44 || !value.match(/^(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$/))
 		return _('Invalid Base64 key string');
 
+	if (value['43'] != "=" )
+		return _('Invalid Base64 key string');
```

Comment:
That looks wrong. `=` is a padding char, not every valid base64 string ends with a `=` char.

Reply:
I did a `wg genkey` and I have not seen any other char then `=`  so I thought this is a must?
Maybe wireguard use this base64 diffrence? We could ask @zx2c4 if he could give us a hint.

Reply:
Seems you're right, apparently a trailing `=` is mandatory:
https://github.com/WireGuard/wireguard-tools/blob/3bf1b64d44fdbe589ac145b61bd4b3aa62b625bf/src/encoding.c#L56

So fine with me then, but please use a numeric subscript (`value[43]` vs. `value['43']`)